ai2thor/util/metrics.py

```diff
@@ -2,6 +2,7 @@
 import json
 import math
 import copy
+import numpy as np
```

Comment:
It doesn't appear that numpy is used in the file.

Reply:
Removed